### ISSUE TITLE
Remove deprecated Options.optStrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4552]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4552|
|Type|task|
|Created|2010-07-06 00:43:47Z|

See the deprecation warning in [twisted/python/usage.py#L301](twisted/python/usage.py#L301), added at least 7 years ago (r9908).

Attachments:

* [optStrings-4552.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f19/f1926f16a63d51f5885a630e21572cf6983a8b40/7aed47ed8b41910d893add793df546994993914c.patch) (956 bytes) - added by thijs on 2011-05-13 15:34:22Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4552 4552
type__task task
reporter__thijs thijs
priority__lowest lowest
milestone__ 
branch__ 
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1278377027000000 1278377027000000
changetime__1305493380000000 1305493380000000
version__None None
owner__thijs thijs
cc__thijs cc__cyli cc__ivank
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|ivank set owner to @thijstriemstra|
|-|-|

1. Please add "Use optParameters instead." to the news file, just in case someone still uses this.

Otherwise, looks good for merging.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [31902]) Remove support for deprecated `Options.optStrings` feature

Author: thijs
Reviewer: ivank
Fixes: [#4552](https://github.com/twisted/twisted/issues/4552)

`Options.optStrings` is no longer supported, having long since been
superceded by `Options.optParameters`.
